docs/src/index.md

```diff
@@ -8,6 +8,76 @@ For more illustrations of its usage in conjunction with other packages, the
+    - [`gadfly.jl`]: High-level plotting library with a syntax akin to that of `ggplot` (in R).
+    - [`vega-lite.jl`]: High-level plotting library with a syntax akin to that of `altair` (in Python).
+- **Data Wrangling**:
+    - [`DataFramesMeta.jl`](https://github.com/JuliaData/DataFramesMeta.jl): A range of convenience functions for DataFrames that augment `select` and `transform` to provide a user experience similar to that provided by `dplyr` in R.
+    - [`Query.jl`](https://github.com/queryverse/Query.jl): Convenience functions with a similar goal to `DataFramesMeta.jl`, but built to work with any tabular library in Julia (more on other tabular libraries below).
+- **And More!**
+    - [`lightgraphs.jl`](https://github.com/JuliaGraphs/LightGraphs.jl): A pure-Julia, high performance network analysis library. Edgelists in DataFrames can be easily converted into Graphs using the [`GraphDataFrameBridge.jl`](https://github.com/JuliaGraphs/GraphDataFrameBridge.jl) package.
+- **IO**:
+    - DataFrames work well with a range of formats, including CSVs (using [`CSV.jl`](https://github.com/JuliaData/CSV.jl)), Stata, SPSS, and SAS files (using [`StatFiles`](https://github.com/queryverse/StatFiles.jl)), and reading (though not writing) parquet files (using [`ParquetFiles`](https://github.com/queryverse/ParquetFiles.jl)).
+
+While not all of these libraries are tightly integrated with DataFrames, because
+DataFrames are essentially collections of aligned Julia vectors, so it is easy
+to (a) pull out a vector for use with a non-DataFrames-integrated library, or (b)
+use `statsmodels` to convert your table into a homogenously-typed matrix.
+
+### Other Julia Tabular Libraries
+
+DataFrames is a great general purpose tool for data manipulation and
+wrangling, but it's not ideal for all applications. For users with more
+specialized needs, consider using:
+
+- [`TypedTables.jl`](https://juliadata.github.io/TypedTables.jl/stable/): Type-stable heterogeneous tables. Useful for improved performance when the structure of your table is relatively stable.
+- [`JuliaDB.jl`](https://juliadata.github.io/JuliaDB.jl/stable/): For users working with data that is too large to fit in memory, we suggest JuliaDB, which offers better performance for large datasets, and can handle out-of-core data manipulations (Python users can think of JuliaDB as the Julia version of `dask`).
+
+Note that most tabular data libraries in the Julia ecosystem (including DataFrames)
+support a common interface (called `Tables.jl`). As a result, some libraries are
+capable or working with a range of tabular data structures, making it easy to
+move between tabular libraries as your needs change. A user of
+[`Query.jl`](https://github.com/queryverse/Query.jl), for example, can use the
+same code to manipulate data in a DataFrame, a TypedTable, or a JuliaDB table.
```

Comment:
Probably not important to mention here, but Query.jl actually doesn't depend on the interface in Tables.jl, the only integration point that is used by Query.jl is https://github.com/queryverse/IteratorInterfaceExtensions.jl :) Tables.jl makes it fairly simple to implement that, but I don't think it is automatic? Not sure, though, what the latest situation is there.